README.md

```diff
@@ -118,16 +118,23 @@ and checkout the version of 2015-01-16:
 git checkout edc6d77bf263695d0ca0e11306d1fecd1be18510
 ```
 
+This Version has a minor error we need to fix.  
+Open `<OpenSGDir>/Source/System/State/Shader/Base/OSGShaderCacheTree.inl` with your favorite editor and go to line 1171. Here a return value is missing. Since `returnValue` is still `NULL` at this point we can use this. Consequently the if should look as follows:
+```c++
+if(pCurrNode == NULL)
+    return returnValue;
+```
+
 Next create a build directory and go into it:
 
 ```sh
 mkdir build && cd build
 ```
 
-First we generate a build environment using `cmake`:
+First we generate a build environment using `cmake` and ensuring the C++11 standard is used:
```

Comment:
Beim 2. Satz-Teil passt die Grammatik nicht. Statt "ensuring" sollte "ensure that" kommen.